src/Nri/Ui/Alert/V3.elm

```diff
@@ -108,6 +108,7 @@ iconContainer styles icon =
         (styles
             ++ [ -- Content positioning
                  Css.marginRight (Css.px 5)
+               , Css.lineHeight (Css.num 1.2)
```

Comment:
🤷‍♀️ Not sure what makes this fall at magic number 1.2, but looks good to me 🤷‍♀️